sbteclipse-plugin/src/sbt-test/sbteclipse/02-contents/project/Build.scala

```diff
@@ -14,7 +14,8 @@ object Build extends Build {
         "org.scala-lang" % "scala-compiler" % "2.10.2",
         "biz.aQute" % "bndlib" % "1.50.0"
       ),
-      retrieveManaged := true
+      retrieveManaged := true,
+      EclipseKeys.createSrc := EclipseCreateSrc.Default - EclipseCreateSrc.Managed
```

Comment:
i think it probably makes more sense to not add this line and instead update the other lines